cypress/support/step_definition/dashboard/dashboard.spec.js

```diff
@@ -1,10 +1,12 @@
 import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+import Page_Navigation from '../../pages/common/pageNavigation'
 import Dashboard from '../../pages/dashboard/pageDashboard'
 
+const pageNavigation = new Page_Navigation()
 const selectDashboard = new Dashboard()
```

Comment:
this should be 'const dashboard' like class name

Reply:
done